tools/custom-action/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:23.04
+
+ENV LANG=en_US.UTF-8
+
+RUN \
+  export DEBIAN_FRONTEND=noninteractive && \
+  apt update -y && \
+  apt install -y locales && \
+  locale-gen en_US.UTF-8 && \
+  sysctl -w kernel.unprivileged_userns_clone=1 && \
+  apt install -y \
+    nodejs \
+    npm \
+    git \
+    sudo \
+    libgtk-3-dev \
+    libnotify-dev \
+    libgconf-2-4 \
+    libnss3 \
+    libxss1 \
+    libasound2 \
+    python3-full \
+    python3-pip \
+    rsync
+
+COPY entrypoint.sh prepare.sh .env /
+
+RUN \
+  chmod +x /entrypoint.sh /prepare.sh /.env && \
+  export $(cat /.env | xargs) && \
```

Comment:
Why this instead of just `source /.env`? I don't think `export` is actually needed, but if it is you can also just put the `export` directives in `.env` itself so it can be sourced directly.

Reply:
Without export, it does not propagate to processes like Python unfortunately.

Reply:
Oh yes, right, we do use it in python. I think we can have
```
export PIP_TREE_SITTER_VERSION=0.20.1
export NPM_MERMAID_CLI_VERSION=9.1.4
export NPM_NAN_VERSION=2.15.0
export NPM_TREE_SITTER_CLI_VERSION=0.20.7
```
though and `source` that file.

Reply:
But that makes it not possible to use the file as .env configuration to Docker container start. It could be good if we preserve dotenv compatibility

Reply:
Ahh I see. What tools actually rely on docker setting the environment globally instead of loading them locally in the script? Can we change them so they don't need it?

Reply:
Trying

Reply:
(Would that also mean it can be moved up to tools/?)